### DISCUSSION TITLE
Mobile chrome Browser with Native App issue by Appium

### DISCUSSION THREAD

**Discussion by qualityStackAILearning** [appium/appium#18950] [Q&A]
Team, Could you help me to advise whether we can pass any app deeplink url in the Android or iOS browser and launch the app after clicking on the Play' icon on the alert popup where we want to close all tabs before launching the same browser for Android and iOS? In this case, I created two appium drivers, but after launching the browser from the second appium driver, I was unable to perform any action on that app from the first appium driver because the first appium driver session closed. Is one Appium driver's capability sufficient to handle mobile Android or iOS browsers and apps? Could you please have a look at the below practise code with a screen shot? I was not able to click the Play with the App' icon button on the Chrome or Safari browser and also not close all tabs once the browser launched because, in the appium inspector, it showed an error message as Interactions are not available for this element.Could you please help me on this? 
scenario :- login with any app -- launch mobile browser but closed all tab before launching browser -- click on "Play with app" --- perform any operation on app
![image (13)](https://github.com/appium/appium/assets/51488902/b238d078-d46b-4fd9-a421-de308ed396ce)
    String deeplink = "https://music.testing.in;
    //to activate chrome browser
    androidDriver.activateApp("com.android.chrome");
    // to click on search box text on browser
    MobileElement Searchbox =(MobileElement)  androidDriver.findElementById("com.android.chrome:id/search_box_text");
    Searchbox.click();
    MobileElement chromeURL =(MobileElement)  androidDriver.findElementById("com.android.chrome:id/url_bar");
    chromeURL.click();
    chromeURL.sendKeys(deeplink);
    androidDriver.pressKey(new KeyEvent(AndroidKey.ENTER));
    ///below two line are failed
    ** **
Thanks you for providing valuable time in advance.

**Comment by jlipps** [comment]
Yes, you should only need a single driver for this case. You can always change the context from web to native.
